Services/Certificate/PRIVACY.md

```diff
@@ -0,0 +1,99 @@
+  to define the nature and extend of this data, the certificate service will store this as part of the final certificate
+  content.
+- Because the replacement of placeholders for final user certificates (possible bulk processing) can be time-consuming, the
+  `Certificate Service` will initially store the issued raw data of the domain event (e.g. `User completes an object`) in an
+  (asynchronous) queue (table: `il_cert_cron_queue`). Once a queue item has been processed, the record will be deleted.
+  The ID of the user account is stored along with the object related data and a datetime information.
+
+## Data being presented
+
+- The certificate service itself provides a `Deck of Card` user interface listing all achieved certificates for the acting
+  user account. The presented data is only object related, enriched with a datetime presentation of the issue date
+  of the respective certificate. Furthermore, a PDF document can be generated (on demand/at runtime). The PDF document
+  contains the textual representation of the [stored data](#data-being-stored).
+- In addition, downloads of certificate PDF documents are provided by consumers of the certificate service. The
+  availability of user interface elements triggering a download of a user certificate and corresponding access checks
+  when the PDF documents are requested depend on the specific consumer. Please check the privacy documentation
+  of the corresponding consumers. 
+
+    Known Consumers:
+
+      - Modules/Course
+      - Modules/StudyProgramme
+      - Modules/Exercise
+      - Modules/Test
+      - Modules/ScormAicc
+      - Modules/CmiXapi
+      - Modules/LTIConsumer
+
+- The certificate provides an internal API. The presentation of this [exported data](#data-being-exported)
+  depends on the consumer. Please check the privacy documentation of the corresponding consumers.
+
+    Known API Consumers:
+
+      - Modules/OrgUnit
+
+## Data being deleted
+
+- If a user account is removed from system, all it's user certificates will be completely deleted from
+  the database.
+
+## Data being exported 
+
+- As mentioned the cerfificate service provides an internal [API](./README.md#api). The consumer is able
+  to request user certificates based on a filter. The API returns an object representation of the
+  [stored database records](#data-being-stored) matching the passed filter.
+- Additionally, the owner of a certificate is able to request a PDF document download of it's achieved
+  certificates (as mentioned above). There might be consumers of the certificate service which provide additional
+  export functionality. Please check the privacy documentation of the corresponding consumers.
+- No other exports of user related data is provided. Only certificate templates are copied or exported if the
+  consuming component (e.g. a course) is copied or a manual export of the certificate is triggered.
```

Comment:
Certificate owners can export PDFs of their Certificates at Achievements > Certificates. 
No other exports of user related data is provided by the Certificate Service itself. Exports may be possible in the object types listed above. Please refer to the respective Privacy.mds.